pdp/portals/gridded_observations.py

```diff
@@ -0,0 +1,61 @@
+'''The pdp.portals.gridded_observations module configures a raster portal
+ which serves gridded climate data used by the VIC model.'''
+
+from pdp import wrap_auth
+from pdp.dispatch import PathDispatcher
+from pdp_util.map import MapApp
+from pdp_util.raster import RasterServer, RasterCatalog, RasterMetadata
+from pdp_util.ensemble_members import EnsembleMemberLister
+
+from pdp.minify import wrap_mini
+from pdp.portals import updateConfig, raster_conf
+
+ensemble_name = 'gridded-obs-met-data'
+url_base = 'gridded_observations'
+
+class GriddedObservationsEnsembleLister(EnsembleMemberLister):
+
+    def list_stuff(self, ensemble):
+        dataset_names = {
+            "ANUSPLIN_CDA_v2012.1": "ANUSPLIN",
```

Comment:
We'll have to wait to hear back from Hydro, but I'd like to omit ANUSPLIN and SYMAP until #37 is fixed.